client/document/index.jsx

```diff
@@ -173,6 +174,30 @@ class Document extends React.Component {
 						} }
 					/>
 
+					{ // Use <script nomodule> to redirect browsers with no ES module
+					// support to the fallback build. ES module support is a convenient
+					// test to determine that a browser is modern enough to handle
+					// the evergreen bundle.
+					isEvergreen && (
+						<script
+							nonce={ inlineScriptNonce }
+							noModule={ true }
```

Comment:
can this just be `nomodule` without the property assignment?

Reply:
The property assignment can be removed, but the capitalisation seems mandatory.